packages/codecs-strings/src/baseX.ts

```diff
@@ -31,7 +31,6 @@ export const getBaseXEncoder = (alphabet: string): VariableSizeEncoder<string> =
             // Handle leading zeroes.
             const [leadingZeroes, tailChars] = partitionLeadingZeroes(value, alphabet[0]);
             if (!tailChars) {
-                bytes.set(new Uint8Array(leadingZeroes.length).fill(0), offset);
```

Comment:
This array is already preallocated with zeros, so I think there's no need to write zeros, unless I'm mistaken.

Reply:
Hmm that's not always true. Someone can use the `write` method of the Encoder using a custom buffer. When they do, they expect the bytes at the given offset to be overwritten with the encoded string bytes. Perhaps we should add a regression test for this.